libraries/cmake/source/lief/CMakeLists.txt

```diff
@@ -0,0 +1,236 @@
+      "${library_root}/src/PE/LoadConfigurations/LoadConfigurationV5.cpp"
+      "${library_root}/src/PE/LoadConfigurations/LoadConfigurationV5.tcc"
+      "${library_root}/src/PE/LoadConfigurations/LoadConfigurationV6.cpp"
+      "${library_root}/src/PE/LoadConfigurations/LoadConfigurationV6.tcc"
+      "${library_root}/src/PE/LoadConfigurations/LoadConfigurationV7.cpp"
+      "${library_root}/src/PE/LoadConfigurations/LoadConfigurationV7.tcc"
+      "${library_root}/src/PE/signature/Attribute.cpp"
+      "${library_root}/src/PE/signature/ContentInfo.cpp"
+      "${library_root}/src/PE/signature/Signature.cpp"
+      "${library_root}/src/PE/signature/SignerInfo.cpp"
+      "${library_root}/src/PE/signature/x509.cpp"
+      "${library_root}/src/PE/signature/OIDToString.cpp"
+      "${library_root}/src/PE/signature/SignatureParser.cpp"
+      "${library_root}/src/PE/signature/RsaInfo.cpp"
+      "${library_root}/src/PE/signature/attributes/GenericType.cpp"
+      "${library_root}/src/PE/signature/attributes/ContentType.cpp"
+      "${library_root}/src/PE/signature/attributes/SpcSpOpusInfo.cpp"
+      "${library_root}/src/PE/signature/attributes/MsCounterSign.cpp"
+      "${library_root}/src/PE/signature/attributes/MsSpcNestedSignature.cpp"
+      "${library_root}/src/PE/signature/attributes/MsSpcStatementType.cpp"
+      "${library_root}/src/PE/signature/attributes/PKCS9AtSequenceNumber.cpp"
+      "${library_root}/src/PE/signature/attributes/PKCS9CounterSignature.cpp"
+      "${library_root}/src/PE/signature/attributes/PKCS9MessageDigest.cpp"
+      "${library_root}/src/PE/signature/attributes/PKCS9SigningTime.cpp"
+      )
+    target_include_directories(thirdparty_lief PRIVATE
+      "${library_root}/src/PE/utils/ordinals_lookup_tables"
+      "${library_root}/src/PE/utils/ordinals_lookup_tables_std"
+      "${library_root}/src/PE/signatures"
+      )
+    file(READ "${library_root}/include/LIEF/PE/enums.inc"      LIEF_PE_ENUMS)
+    file(READ "${library_root}/include/LIEF/PE/structures.inc" LIEF_PE_STRUCTURES)
+    configure_file(
+      "${library_root}/include/LIEF/PE/enums.hpp.in"
+      "${library_root}/include/LIEF/PE/enums.hpp"
+    @ONLY)
+    configure_file(
+      "${library_root}/include/LIEF/PE/Structures.hpp.in"
+      "${library_root}/include/LIEF/PE/Structures.hpp"
+    @ONLY)
+    configure_file("${library_root}/include/LIEF/config.h.in" "${library_root}/include/LIEF/config.h")
+    configure_file(
+      "${library_root}/src/compiler_support.h.in"
+      "${library_root}/src/compiler_support.h"
+    @ONLY)
+
+    target_include_directories(thirdparty_lief PUBLIC
+     "${CMAKE_CURRENT_SOURCE_DIR}/config/windows"
+    )
+  
+    target_compile_definitions(thirdparty_lief PRIVATE NOMINMAX)
+    # Used for the 'and', 'or' ... keywords - See: http://www.cplusplus.com/reference/ciso646/
+    target_compile_options(thirdparty_lief PUBLIC /FIiso646.h)
+    set_property(TARGET thirdparty_lief PROPERTY LINK_FLAGS /NODEFAULTLIB:MSVCRT)
```

Comment:
This is unnecessary in our case, please remove it.
```suggestion
```

Reply:
good to know. Removed